File: {{ cookiecutter.repo_name }}/remote/src/release/java/{{ cookiecutter.package_dir }}/remote/NetworkSettings.java
package {{ cookiecutter.package_name }}.remote;

import dagger.Module;
import dagger.Provides;
import okhttp3.logging.HttpLoggingInterceptor.Level;

@Module
public abstract class NetworkSettings {
    @Provides
    static Level providesLoggingLevel() {
        return Level.NONE;
    }
}

File: {{ cookiecutter.repo_name }}/remote/src/debug/java/{{ cookiecutter.package_dir }}/remote/NetworkSettings.kt
package {{ cookiecutter.package_name }}.remote;

import dagger.Module
import dagger.Provides
import okhttp3.logging.HttpLoggingInterceptor.Level

@Module
internal object NetworkSettings {
    @Provides
    @JvmStatic
    fun providesLoggingLevel(): Level = Level.BODY
}